src/hooks/catalog/useCatalog.ts

```diff
@@ -674,11 +674,11 @@ const useCatalogState = () =>
     {
         const parser = event.getParser();
 
-        setFurniLimit(parser._Str_15864);
-        setMaxFurniLimit(parser._Str_24094);
-        setSecondsLeft(parser._Str_3709);
+        setFurniLimit(parser.furniLimit);
```

Comment:
This change shouldn't make it into this PR either since it's also related to a different PR

Reply:
Made ! 👍